src/psyclone/dynamo0p3.py

```diff
@@ -7131,155 +7406,189 @@ def create_halo_exchanges(self):
                     # or not
                     self._add_halo_exchange(halo_field)
 
+    @property
+    def start_expr(self):
+        '''
+        :returns: the PSyIR for the lower bound of this loop.
+        :rtype: :py:class:`psyclone.psyir.Node`
+
+        '''
+        inv_sched = self.ancestor(InvokeSchedule)
+        sym_table = inv_sched.symbol_table
+        loops = inv_sched.loops()
+        posn = loops.index(self)
+        lbound = sym_table.lookup_with_tag(f"loop{posn}_start")
+        return Reference(lbound)
+
+    @property
+    def stop_expr(self):
+        '''
+        :returns: the PSyIR for the upper bound of this loop.
```

Comment:
Again, either choose return or returns for this PR in this file please.

Reply:
Left.